install/gcp/installer/util/gcloud.py

```diff
@@ -51,6 +52,23 @@ def get_gcloud_info():
             sys.exit(1)
     return project_id, authed_user, is_devshell
 
+def activate_service_account(authed_user, key_path):
+    """Activate the service account with gcloud
```

Comment:
Add a period at the end of this sentence.  This will be in the generated pydoc that we put online.

Reply:
Done